src/System.Text.Json/tests/WritableJsonApiTests.cs

```diff
@@ -0,0 +1,671 @@
+        /// </summary>
+        [Fact]
+        public static void TestCreatingJsonObjectNewMethods()
+        {
+            var developer = new JsonObject
+            {
+                { "name", new JsonString("Kasia") },
+                { "age", new JsonNumber(22) },
+                { "is developer", new JsonBool(true) },
+                { "null property", new JsonNull() }
+            };
+        }
+
+        /// <summary>
+        /// Creating and retriving different numeric values
+        /// </summary>
+        [Fact]
+        public static void TestNumerics()
+        {
+            double PI = 3.14159265359;
+            var circle = new JsonObject
+            {
+                { "radius", 1 },
+                { "length", 2*PI },
+                { "area", PI }
+            };
+
+            JsonNumber bigConstantBoxed = Int64.MaxValue;
+            long bigConstant = bigConstantBoxed.GetInt64();
+
+            var smallValueBoxed = new JsonNumber(17);
+            smallValueBoxed.TryGetInt16(out short smallValue);
+        }
+
+        /// <summary>
+        /// Creating nested Json object
+        /// </summary>
+        [Fact]
+        public static void TestCreatingNestedJsonObject()
+        {
+            var person = new JsonObject
+            {
+                { "name", "John" },
+                { "surname", "Smith" },
+                {
+                    "phone numbers", new JsonObject()
+                    {
+                        { "work", "123-456-7890" },
+                        { "home", "123-456-7890" }
+                    }
+                },
+                {
+                    "addresses", new JsonObject()
+                    {
+                        {
+                            "office", new JsonObject()
+                            {
+                                {  "address line 1", "One Microsoft Way" },
+                                {  "city" , "Redmond" } ,
+                                {  "zip code" , 98052 } ,
+                                {  "state" , (int) AvailableStateCodes.WA }
+                            }
+                        },
+                        {
+                            "home", new JsonObject()
+                            {
+                                {  "address line 1", "Pear Ave" },
+                                {  "address line 2", "1288" },
+                                {  "city" , "Mountain View" } ,
+                                {  "zip code" , 94043 } ,
+                                {  "state" , (int) AvailableStateCodes.CA }
+                            }
+                        }
+                    }
+                }
+            };
+        }
+
+        /// <summary>
+        /// Defining as KeyValuePair value
+        /// </summary>
+        [Fact]
+        public static void TestAssignmentDefinition()
+        {
+            var employee = EmployeesDatabase.GetNextEmployee().Value;
```

Comment:
The corefx style guidelines only allow `var` when it's used with a `new`, `as`-cast, or hard-cast expression.

https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md

Reply:
OK, I'll change it.